repo/packages/Q/quobyte/2/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
`command.json` has been deprecated with version 3.0 in favor of the `.cli` property of `resource.json`
please see: https://github.com/mesosphere/universe#commandjson